src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -0,0 +1,107 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.PatientCaptureVitalsPage;
+import org.openmrs.uitestframework.test.TestData;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.junit.Assert.assertTrue;
+
+public class VitalsAndTriagingSteps extends Steps {
+
```

Comment:
Added an empty `LN-18` with reference to the artifacts [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/472#issuecomment-867370006)